src/components/renderer/PageFormElements.tsx

```diff
@@ -37,6 +37,28 @@ function PageFormElements({
   onChange,
   setFormSubmission,
 }: Props) {
+  // Effect responsible for focusing the first element on a page when the next button is focused and used to nav forwards a page
+  React.useEffect(() => {
+    if (isActive) {
+      console.log(`${pageElement.label} page became active`)
+      const currentFocusedElement = document.activeElement
+      if (currentFocusedElement?.getAttribute('data-nav-button') === 'next') {
+        const selector =
+          'a:not([disabled]), button:not([disabled]), input:not([disabled]), select:not([disabled]), [tabindex]:not([disabled]):not([tabindex="-1"])'
```

Comment:
does *:not([disabled]) work?

Reply:
That will give us things that we can't focus.